app/helpers/reactjs_helper.rb

```diff
@@ -70,7 +70,7 @@ def locale_js_tags
     ::Foreman::Plugin.all.filter_map do |plugin|
       domain = plugin.gettext_domain
       if domain && (FastGettext.translation_repositories[domain]&.available_locales || []).include?(locale)
-        javascript_include_tag("locale/#{locale}/#{domain}")
```

Comment:
I've just noticed that if I have preferred language in the browser, e.g. en_GB, but a plugin doesn't have js file for that, I won't be able to even log in to change the application language :/

![ScreenShot-1688133568083](https://github.com/theforeman/foreman/assets/32508194/712b0900-8a03-4b36-9684-b8d4170fe154)

Reply:
Something I did suspect when reviewing it:
https://github.com/theforeman/foreman/pull/9691#discussion_r1190043816

I think that got lost somewhere in the various iterations the patch went through.